gallery/src/components/NxDrawer/NxDrawerPage.tsx

```diff
@@ -113,6 +114,27 @@ export default function NxDrawerPage() {
                   "normal" drawers are 520px wide, and "narrow" drawers are 348px wide.
                 </NxTable.Cell>
               </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>closeBtnDisabled</NxTable.Cell>
+                <NxTable.Cell>boolean</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>false</NxTable.Cell>
+                <NxTable.Cell>
+                  This prop, when set to true, will disable the close button on the drawer and disable
+                  the drawer from being closed.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>closeBtnDisabledTooltip</NxTable.Cell>
```

Comment:
I'm thinking it would be simpler and more powerful if it was just `closeBtnTooltip`, and could be specified regardless of whether the close button is disabled. Then calling code can set it conditionally if they only want it to change when the button is disabled.

Reply:
> I'm thinking it would be simpler and more powerful if it was just `closeBtnTooltip`, and could be specified regardless of whether the close button is disabled. Then calling code can set it conditionally if they only want it to change when the button is disabled.

I initially started with `closeBtnTooltip`, but changed it later. Changed prop name here: 254fc8c9315ef74709ddfd45e97d24364269c794

Changed functionality, tests and docs here: add89ebdd9fe5b42c0e743fdd611b6cd077e737f